Makefile

```diff
@@ -138,8 +138,9 @@ vet: $(SOURCES)
 # -ST1020 too many wrong comments on exported functions to fix right away
 # -ST1021 too many wrong comments on exported functions to fix right away
 # -ST1022 too many wrong comments on exported functions to fix right away
+# -SA1019 using non-cryptographic rand functions
 staticcheck: $(SOURCES)
-	staticcheck -checks "all,-ST1000,-ST1003,-ST1012,-ST1020,-ST1021" $(PACKAGES)
+	staticcheck -checks "all,-ST1000,-ST1003,-ST1012,-ST1020,-ST1021,-SA1019" $(PACKAGES)
```

Comment:
I think it would be better to add this exception only at the place where it is. Otherwise we may miss introducing this in a new place in the future.

Reply:
Or even better ofc. is to fix the cases where the deprecated function is used. :)